docs/architecture/orders.rst

```diff
@@ -4,50 +4,33 @@ Order Management
 Orders are created after customers complete the checkout process. The `Order` object itself contains only general information about the customer's order.
 
 
-Delivery Group
---------------
-
-The delivery group represents a group of ordered items. By default, groups are created along with an order by splitting the cart into parts, depending whether a product requires shipping or not.
+Fulfillment
+-----------
 
-Most of the order management actions are taken on the delivery groups and include changing their statuses.
+The fulfillment represents a group of shipped items with corresponding tracking number. Fulfillments are created by a shop operator to mark selected products in an order as fulfilled.
 
-There are three possible delivery group statuses:
+There are two possible fulfillment statuses:
```

Comment:
Shouldn't there exist a third status to indicate which groups are already dispatched?

Reply:
I came from the assumption, that we consider fulfilled group as dispatched. This is why user also is asked for tracking number, when fulfilling the items.

Reply:
How does that work with packing slips? The usual flow for a warehouse (where packing slips are used) is:

1. dispatcher creates a shipment group (fulfillment)
2. warehouse prints a packing slip
3. warehouse collects and packages items
4. warehouse dispatches the order (this is usually the moment you know the tracking number)

Reply:
For now we could leave this as it is, cause it's quite simple. We can discuss further changes in the flow and open new PR if necessary.